### DISCUSSION TITLE
Adding new content

### DISCUSSION THREAD

**Discussion by canadianeagle** [Vagr9K/gatsby-advanced-starter#155] [Q&A] [answered]
Hello again,

I'm trying to do things the right way (via shadowing and the like).
I want to add new content but I can't get my new content to show - I added the content folder in my shadowed folder
`src\gatsby-theme-amaranth\content` and even tried `src\gatsby-theme-amaranth\gatsby-theme-advanced\content` but nothing is working.

**So my actual question:**
Do I use the folder `themes\advanced\content` and add my content there? Is this the right way to do things without having to mess anything up in the future when I update? 

Thanks again :)

T

**Comment by Vagr9K** [comment, marked as answer]
Hey!

Adding content isn't related to shadowing. The base content is just a fallback for development purposes.

You just need to configure `contentDir` and `assetDir` as per configuration document here: https://github.com/Vagr9K/gatsby-advanced-starter/tree/master/themes/amaranth#configuration.

The easiest option would be to set `contentDir` as `content` and create a `content` folder (in the starter root) with MDX/MD posts inside.

**Reply by canadianeagle** [reply]
Thanks so much :))
